package.json

```
@@ -3,7 +3,7 @@
     "version": "1.0.1",
     "description": "Midi Node",
     "dependencies": {
-        "midi": "~0.9.5"
+        "midi": "~1.0.0"
```

```suggestion
        "midi": "~1.0.4"
```

See https://github.com/justinlatimer/node-midi/releases for new releases.

---

Considering this PR is almost a year old this is a fair suggestion :D